pkg/admissioncontroller/webhook/admission/authorizationconfig/add_test.go

```diff
@@ -237,6 +237,19 @@ authorizers:
 				test(admissionv1.Update, shoot, newShoot, true, statusCodeAllowed, "referenced authorization configuration is valid", "")
 			})
 
+			It("Kubeconfig secret refs is changed (UPDATE)", func() {
+				returnedCm := corev1.ConfigMap{
+					Data: map[string]string{"config.yaml": validAuthorizationConfiguration},
+				}
+				newShoot := shoot.DeepCopy()
+				newShoot.Spec.Kubernetes.KubeAPIServer.StructuredAuthorization.Kubeconfigs[0].AuthorizerName = "does-not-exist"
+				mockReader.EXPECT().Get(gomock.Any(), client.ObjectKey{Namespace: shootNamespace, Name: configMapName}, gomock.AssignableToTypeOf(&corev1.ConfigMap{})).DoAndReturn(func(_ context.Context, _ client.ObjectKey, cm *corev1.ConfigMap, _ ...client.GetOption) error {
+					*cm = returnedCm
+					return nil
+				})
+				test(admissionv1.Update, shoot, newShoot, true, statusCodeAllowed, "provided invalid authorization configuration: must provide kubeconfig secret name reference for webhook authorizer \"webhook\" in shoot fake-cm-namespace/fake-shoot-name", "")
```

Comment:
```suggestion
				test(admissionv1.Update, shoot, newShoot, false, statusCodeInvalid, "provided invalid authorization configuration: must provide kubeconfig secret name reference for webhook authorizer \"webhook\" in shoot fake-cm-namespace/fake-shoot-name", "")
```

And should be moved to the `Deny` context

Reply:
Done, sorry. Pushed too fast 🚤 :)